requirements.txt

```diff
@@ -1,5 +1,5 @@
-rasa_nlu[tensorflow]==0.13.7
-rasa_core==0.13.0a4
+rasa_nlu[tensorflow]~=0.14.4
```

Comment:
0.14.6 and 0.13.6 for nlu/core respectively please :D 
also do you mind adding the beautiful4soup/requests/geopy and pandas requirements here

Reply:
could we possibly minor release vova's form evaluation fix in core and then run on that?

Reply:
i believe so -- can you tag vova in channel and ask? we should also see if anyone else wants to put anything into a new minor